src/Documentation/Abstract_Doc_Command.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Tribe\CLI\Documentation;
+
+use WP_CLI;
+
+abstract class Abstract_Doc_Command extends \WP_CLI_Command {
+	protected $plugin_dir;
+
+	/**
+	 * Parses out the passed plugin name
+	 *
+	 * @param array $args
+	 *
+	 * @return mixed
+	 *
+	 * @throws WP_CLI\ExitException
+	 */
+	protected function parse_plugin( array $args ) {
+		$plugin = reset( $args );
+		if ( ! $plugin ) {
+			WP_CLI::error( 'You must specify a plugin' );
+			die;
```

Comment:
Here and elsewhere: by default, `WP_CLI::error()` will `exit`, so no need to explicitly `die` (though if you feel it makes things clearer, that's legit).